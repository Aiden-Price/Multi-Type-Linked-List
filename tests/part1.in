NEW 3
a
b
c
NEW noletter
NEW 
 NEW
 NEW 3
TYPE 0
TYPE 2
REMOVE 0
NEW 10
b
12
2e-3
32.34343
22.12
2.0
test
r12e
5
4