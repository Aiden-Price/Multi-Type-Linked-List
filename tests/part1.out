List 0: a -> b -> c
INVALID COMMAND: NEW
INVALID COMMAND: NEW
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
char -> char -> char
INVALID COMMAND: TYPE
List 0 has been removed.

Number of lists: 0
List 1: b -> 12 -> 0.00 -> 32.34 -> 22.12 -> 2.00 -> test -> r12e -> 5 -> 4
